ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V1__Append_only_schema.sha256

```diff
@@ -1 +1 @@
-fe940b22140c0a1da0a3e54c0087c8872828e4cb08bdcc30bbaa1db84f98b62b
+d4881b98b80ff99eea6cd98b48be75cee9f006da63e3850cf28b300a11cfc54e
```

Comment:
I thought the Oracle migrations had shipped and we now need to add new migrations?

Reply:
Reverted the Oracle schema white space changes in 22cd962, they are not even worth the time pondering if we need new migrations or not.